<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QJDTextBrowser</class>
 <widget class="QMainWindow" name="QJDTextBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QJDTextBrowser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/images/images/gnome-logout.png</normaloff>:/new/images/images/gnome-logout.png</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>txt1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>txt2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Copy"/>
    <addaction name="separator"/>
    <addaction name="action_All"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Copy"/>
   <addaction name="separator"/>
   <addaction name="action_Exit"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/editcopy.png</normaloff>:/images/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_All">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
